src/middlewares/store.coffee

```diff
@@ -9,5 +9,10 @@ module.exports = (req, res, next) ->
       .then (data) ->
         transforms.store.transform data
   )
+  res.promises.push(
+    api.get "stores/#{req.app.get 'store_id'}/instagram/media/recent"
```

Comment:
It will, but for this to behave like in production and not render the tags if you don't have instagram connected, we need to query this on the gulp-tictail backend too.